sys/include/net/gnrc/netreg.h

```diff
@@ -284,8 +296,11 @@ static inline void gnrc_netreg_entry_init_cb(gnrc_netreg_entry_t *entry,
  *
  * @param[in] type      Type of the protocol. Must not be < GNRC_NETTYPE_UNDEF or
  *                      >= GNRC_NETTYPE_NUMOF.
- * @param[in] entry     An entry you want to add to the registry with
- *                      gnrc_netreg_entry_t::pid and gnrc_netreg_entry_t::demux_ctx set.
+ * @param[in] entry     An entry you want to add to the registry. This needs to
```

Comment:
I find this addressing the reader using `you` a bit uncommon for documentation - I know it was there before, but still this about improving docu, right 😉 Hence, propose to change to something like: `New registry entry to be added.`